python/cudf/cudf/tests/test_serialize.py

```diff
@@ -233,3 +233,18 @@ def test_serialize_all_null_string():
 
     recreated = cudf.Series.deserialize(*gd_series.serialize())
     assert_eq(recreated, pd_series)
+
+
+def test_serialize_named_series():
```

Comment:
It also might be nice to add a test for the dask_cudf example that exposed the issue in https://github.com/rapidsai/cudf/issues/3913

Reply:
Okay - Not sure if it will be covered by CI (since it requires ucx), but I added it.